### PR TITLE
fix: adjust cell coord to totals position when clikcing to focus

### PR DESCRIPTION
**Pull request by haxxmaxx**
SImply need to take the totalsPosition into accounce, since it means that the row index should be one more + 1 if it is on top

### CODE REVIEW

File: src/table/utils/handle-accessibility.js [hunk 1]
```diff
@@ -34,9 +34,10 @@ export const removeAndFocus = (newCoord, rootElement, setFocusedCellCoord, keybo
   keyboard.enabled && keyboard.focus();
 };
 
-export const handleClickToFocusBody = (cell, rootElement, setFocusedCellCoord, keyboard) => {
+export const handleClickToFocusBody = (cell, rootElement, setFocusedCellCoord, keyboard, totalsPosition) => {
   const { rawRowIdx, rawColIdx } = cell;
-  removeAndFocus([rawRowIdx + 1, rawColIdx], rootElement, setFocusedCellCoord, keyboard);
+  const firstBodyRowIdx = totalsPosition === 'top' ? 2 : 1;
```
Comment (LiKang6688) [inline src/table/utils/handle-accessibility.js:39]: the variable name `firstBodyRowIdx` does looks confused to me.

Comment (haxxmaxx) [reply src/table/utils/handle-accessibility.js:39]: it is the index that the first row has, when you take header and totals into account. the `rawRowIdx` is the index in the body, then we need to add 1 or 2 to compensate. Do you have any other suggestions?

Comment (LiKang6688) [reply src/table/utils/handle-accessibility.js:39]: how about
```
const cellRowIdx = totalsPosition === 'top' ? rawRowIdx + 2 : rawRowIdx + 1;
  removeAndFocus([cellRowIdx, rawColIdx], rootElement, setFocusedCellCoord, keyboard);
```

**Review by LiKang6688 — APPROVED** [review]

File: src/table/utils/handle-accessibility.js (hunk 1 above)
Comment (haxxmaxx) [reply src/table/utils/handle-accessibility.js:39]: sure, I might call it `adjustedRowIdx` though

**Pull request merged**